velox/docs/functions/presto/conversion.rst

```diff
@@ -457,6 +457,7 @@ Valid examples
   SELECT cast(infinity() as varchar); -- 'Infinity'
   SELECT cast(true as varchar); -- 'true'
   SELECT cast(timestamp '1970-01-01 00:00:00' as varchar); -- '1970-01-01T00:00:00.000'
+  SELECT cast(cast(22.51 as DECIMAL(4, 2)) as varchar); -- '22.51'
```

Comment:
Let's also add examples with negative numbers and numbers like 0.123 and 1.00.

Reply:
Updated.